mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReprocessingContextInformationDTO.java

```diff
@@ -58,15 +58,27 @@ public String getMailboxId() {
     }
 
     public static class ReprocessingContextInformationForErrorRecoveryIndexationTask extends ReprocessingContextInformation {
-        public static final AdditionalInformationDTOModule<ReprocessingContextInformationForErrorRecoveryIndexationTask, ReprocessingContextInformationDTO> serializationModule(MailboxId.Factory mailboxIdFactory) {
+
+        public static class DTO extends ReprocessingContextInformationDTO {
+
+            DTO(@JsonProperty("type") String type,
+                @JsonProperty("successfullyReprocessedMailCount") int successfullyReprocessedMailCount,
+                @JsonProperty("failedReprocessedMailCount") int failedReprocessedMailCount,
+                @JsonProperty("failures") List<ReindexingFailureDTO> failures,
+                @JsonProperty("timestamp") Instant timestamp) {
+                super(type, successfullyReprocessedMailCount, failedReprocessedMailCount, failures, timestamp);
+            }
+        }
+
+        public static final AdditionalInformationDTOModule<ReprocessingContextInformationForErrorRecoveryIndexationTask, DTO> serializationModule(MailboxId.Factory mailboxIdFactory) {
             return DTOModule.forDomainObject(ReprocessingContextInformationForErrorRecoveryIndexationTask.class)
-                .convertToDTO(ReprocessingContextInformationDTO.class)
+                .convertToDTO(DTO.class)
```

Comment:
ReprocessingContextInformationForErrorRecoveryIndexationTask.DO?

Reply:
no?

Reply:
Is it the same question I answered here https://github.com/linagora/james-project/pull/2784#discussion_r338094773 or do I miss something?

Reply:
DTO is an inner class, so ok to name it DTO. But when you use this name, why not using the full name to explicit the fact that it is a specific DTO?